isacc_messaging/models/isacc_patient.py

```diff
@@ -32,12 +32,31 @@ def __repr__(self):
     def active_patients():
         """Execute query for active patients
 
+        NB, returns only patients with active set to true
+        """
+        response = HAPI_request('GET', 'Patient', params={
+            "active": "true"
+        })
+        return response
+
+    @staticmethod
+    def all_patients():
+        """Execute query for all patients
+
         NB, until status is set on all patients, queries for
         any status/active value will skip those without a value.
         """
         response = HAPI_request('GET', 'Patient')
```

Comment:
@Filienko Do you know if this does indeed return **all** the patients or does it return paginated results?  For instance, I think we had to loop through all pages to get at all the CommunicationRequest resources when sending out email notifications (see [here](https://github.com/uwcirg/isacc-messaging-service/blob/00473047a1dd25e203b01645638d0276cf676c47/isacc_messaging/api/isacc_record_creator.py#L422)).  I am not certain if we need to consider the same, i.e. pagination, here.